fpga/src/bootrom/platform.h

```diff
@@ -1,2 +0,0 @@
-#define DRAM_BASE 0x80000000
```

Comment:
Why don't we need this file anymore?

Reply:
We actually also have a Genesys 2 board. Everything is tested on our Ubuntu 18.04, with Genesys2 and Nexys Video board.

`fpga/src/bootrom/platform.h` is moved to `fpga/src/bootrom/default/platform.h`. Now if no `BOARD` environment variable is set when doing `make all` inside `fpga/src/bootrom`, the files in `default` directory is used.

I found that `DRAM_BASE` is set to 0x8000_0000 in so many places (not only in bootrom,  but also Ariane-SDK) that making this a variable don't make much difference; no code in bootrom is using this macro anyway.

One thing I also want to mention is that if I generate the boorom with default `ariane.dts`, the clock used for linux is not correct; `sleep 10` will only take roughly 6 seconds. I changed `timebase-frequency` in device tree to be half of cpu frequency in the Nexys Video port and it seems to work. I suggest to make the same change to the default `ariane.dts`.